package.json

```diff
@@ -100,6 +100,7 @@
     "zone.js": "^0.7.7"
   },
   "dependencies": {
-    "date-fns": "^1.28.0"
+    "date-fns": "^1.28.0",
+    "lodash": "^4.17.4"
```

Comment:
sorry but third party library here is a overkill for a simple object manipulations.